src/components/card/card.css

```diff
@@ -1,3 +1,35 @@
+.sm-card:hover {
```

Comment:
UI Suggestion: you could give a light background color to card to distinguish it from the app background

![image](https://user-images.githubusercontent.com/91944618/161439684-6e1be8c8-99ff-4746-aa69-7d12463513da.png)